### PR TITLE
refactor: remove esbuild

### PR DESCRIPTION
**Pull request by himself65**
https://github.com/dai-shi/waku/pull/132#discussion_r1370887758

### CODE REVIEW

File: packages/create-waku/tsconfig.json [hunk 1]
```diff
@@ -5,8 +5,10 @@
     "outDir": "./dist",
     "module": "NodeNext",
     "moduleResolution": "nodenext",
-    "lib": ["ESNext", "DOM", "DOM.Iterable"],
-    "tsBuildInfoFile": "./dist/.tsbuildinfo"
+    "types": [],
```
Comment (dai-shi) [inline packages/create-waku/tsconfig.json:8]:
```suggestion
```

**Review by dai-shi — COMMENTED** [review]
Nice, we thought about using `swc` but `tsc` would work too. Would you be interested in doing it with `swc`, like `packages/waku`?

**Comment by himself65** [comment]
> Would you be interested in doing it with swc, like packages/waku?

I tried, that is impossible if the project both output cjs and esm, since tsconfig will check `require` in cjs, `import.meta.url` in esm. they cannot both together

**Comment by dai-shi** [comment]
Interesting. I wasn't aware of it.

Did you try the cli locally to see if works as expected? Can I merge and just release it?

**Comment by himself65** [comment]
I tried it, works

**Review by dai-shi — APPROVED** [review]
Thanks so much.

**Pull request merged**